txclib/project.py

```diff
@@ -715,6 +733,8 @@ def push(self, source=False, translations=False, force=False,
                 # Push source file
                 try:
                     logger.warning("Pushing source file (%s)" % sfile)
+                    stats = self._get_stats_for_resource()
```

Comment:
@SofiaMargariti moving this line in try block causes us problem. This method used to raise an Exception when no credentials found for host and exit. Unfortunately the try block handles AuthenticationError and SSLError and continues execution for all other exceptions. I think is to remove the else condition of try block and handle only the exception types that we want. Thought this might be risky